src/style.css

```
@@ -1 +1,64 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.image {
+  display: flex;
+  margin-top: 10px;
+  flex-shrink: 0;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  padding: 20px 0;
+  flex-shrink: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.link {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  justify-content: center;
+}
+
+.link:hover {
+  padding-top: 23px;
+  color: #00acdc;
+}
+
+.is-active {
+  padding-top: 23px;
+  color: #00acdc;
+}
```

Looks like you can delete padding-top

```suggestion
.is-active {
  color: #00acdc;
}
```